client/src/components/Home/index.js

```diff
@@ -249,11 +249,11 @@ export default function Home(props) {
               <p>
                 <a href="#" className="header-link">
                   {t("unsigned-home-demo.user-guides")}
-                </a>{" "}
+                </a>{"https://docs.rocket.chat/guides/user-guides"}
                 |{" "}
                 <a href="#" className="header-link">
                   {t("unsigned-home-demo.release-notes")}
-                </a>
+                </a>{"https://github.com/RocketChat/Rocket.Chat/releases"}
```

Comment:
The `{"https://github.com/RocketChat/Rocket.Chat/releases"}` part should be removed here. (It renders as text currently)

Reply:
ok